src/components/SpaceProposalVotesModal.vue

```diff
@@ -0,0 +1,149 @@
+<script setup lang="ts">
+import { ref, toRefs, watch } from 'vue';
+import { useScroll } from '@vueuse/core';
+import { useProposalVotes } from '@/composables';
+import {
+  ExtendedSpace,
+  Proposal,
+  Vote,
+  SpaceStrategy
+} from '@/helpers/interfaces';
+
+const props = defineProps<{
+  space: ExtendedSpace;
+  proposal: Proposal;
+  strategies: SpaceStrategy[];
+  userVote: Vote | null;
+  open: boolean;
+}>();
+
+defineEmits(['close']);
+
+const votesScrollWrapper = ref<HTMLElement | null>(null);
+const votesQuery = ref('');
+
+const {
+  loadVotes,
+  sortedVotes,
+  loadMore,
+  loadingMore,
+  loadedVotes,
+  loadMoreVotes,
+  profiles,
+  clearVotes,
+  votesNotFound,
+  isENS,
+  resolvingENS,
+  wrongENS
+} = useProposalVotes(props.proposal, 20, props.userVote, votesQuery);
+
+const { arrivedState } = useScroll(votesScrollWrapper, {
+  offset: {
+    bottom: 60 * 10
+  }
+});
+
+const { bottom } = toRefs(arrivedState);
+
+watch(
+  () => bottom.value,
+  to => {
+    if (props.open && to) {
+      loadMore(loadMoreVotes);
+    }
+  },
+  {
+    immediate: false
+  }
+);
+
+watch(
+  () => props.open,
+  to => {
+    if (to) {
+      loadVotes(20);
+    } else {
+      clearVotes();
+    }
+  }
+);
+</script>
+
+<template>
+  <BaseModal :open="open" @close="$emit('close')">
+    <template #header>
+      <div
+        class="flex flex-col content-center items-center justify-center gap-x-4"
+      >
+        <h3>{{ $t('votes') }}</h3>
+        <BaseSearch
+          v-model="votesQuery"
+          :placeholder="$t('searchPlaceholder')"
+          :modal="true"
+          :focus-on-mount="true"
+          class="min-h-[60px] w-full flex-auto px-3 pb-3"
+        >
+          <template
+            v-if="isENS && (resolvingENS || wrongENS)"
+            #after="{ clearInput }"
+          >
+            <LoadingSpinner
+              v-if="isENS && resolvingENS && !wrongENS"
+              class="pb-[3px]"
+            />
+            <i-ho-x
```

Comment:
This doesn't seem so obvious, I think it would be better to always show `no results found` when something was entered into the search input but no vote or ens address was found.